libgalois/include/katana/analytics/ClusteringImplementationBase.h

```diff
@@ -71,16 +77,16 @@ struct ClusteringImplementationBase {
   template <typename EdgeWeightType>
   void FindNeighboringClusters(
       const Graph& graph, GNode& n,
-      std::map<uint64_t, uint64_t>& cluster_local_map,
-      std::vector<EdgeTy>& counter, EdgeTy& self_loop_wt) {
-    uint64_t num_unique_clusters = 0;
+      std::map<CommunityIdTy, CommunityIdTy>* cluster_local_map,
+      std::vector<EdgeTy>* counter, EdgeTy* self_loop_wt) {
+    CommunityIdTy num_unique_clusters = 0;
```

Comment:
this should be uint64_t, as it was before. as it is computng the number of clusters

Reply:
Shouldn't the maximum number of clusters be equal to the number of nodes of the original graph?
Our graph is generally using uint32 for the number of nodes and uint64 for the number of edges.

Reply:
you are right

Reply:
Also, the current CommunityID is uint64_t